### PR TITLE
RoomFind Beautification

### PR DESCRIPTION
**Pull request by tsartin**
Updating and beautifying mmp.RoomFind. 

Updating to make RoomFind's output more readable. It also now only shows 30 results by default and sorts by area in alphabetical order.

You can still click the room number to go but to show the path you know click the area instead of having an additional 'show path' text.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -7730,11 +7730,32 @@ end</script>
   end
 end
 
-function mmp.roomFind(query)
+function mmp.roomFind(query, lines)
   if query:ends('.') then
     query = query:sub(1, -2)
   end
+  local defaultLine = 30 --could this to a setting
```
Comment (vadi2) [inline mudlet-mapper.xml:7737]:
```suggestion
  local defaultLine = 30 --could move this to a setting
```

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -7730,11 +7730,32 @@ end</script>
...
   local result = mmp.searchRoom(query)
+  if lines == 'all' then
+    lines = table.size(result)
+  end
+  lines = (lines ~= '') and tonumber(lines) or defaultLine 
+  
+  --create a new table (_nt) with keys and add areas to the table
+  local _nt = {}
```
Comment (vadi2) [inline mudlet-mapper.xml:7745]: Could you give this a better name? We don't use the underscore for private variables either

**Review by vadi2 — APPROVED** [review]
Looks good to me, thanks for the improvement!

@keneanung and @ThaisenLV want to review?

File: mudlet-mapper.xml (hunk 2 above)
Comment (tsartin) [reply mudlet-mapper.xml]: renamed and uploaded!

**Comment by QwindorRousseau** [comment]
Remember to update the wiki if needed!!!😁

**Review by vadi2 — APPROVED** [review]

**Pull request merged**